docs/advanced_documentation/native-data-interface.md

```diff
@@ -98,10 +98,11 @@ For example, the following update dataset will set `from_status` of the line #5
 and keep the `to_status` as unchanged (change not available).
 
 ```python
-from power_grid_model import power_grid_meta_data
+from power_grid_model import ComponentType, DataType, power_grid_meta_data
 import numpy as np
 
-line_update = np.empty(shape=1, dtype=power_grid_meta_data['update']['line']['dtype'])
+line_update = np.empty(shape=1, dtype=power_grid_meta_data[DataType.update][ComponentType.line]['dtype'])
+# line_update = np.empty(shape=1, dtype=power_grid_meta_data['update']['line']['dtype']) is supported as well.
```

Comment:
Remove commented out lines.

Reply:
@Jerry-Jinfeng-Guo I left the commented line on purpose to explain that the old format is still supported. It was a suggestion by @mgovers